src/get-info.js

```diff
@@ -71,6 +71,9 @@ var Getters = {
             if (video.meta.blocked) {
                 meta.restricted = video.meta.blocked;
             }
+            if (video.meta.ytRating) {
+                meta.rating = video.meta.ytRating;
```

Comment:
Any particular reason to drop the `yt` prefix when cloning this property from the mediaquery model to the legacy cytube model?

Reply:
Not really. Having it as ytRating is probably a little clearer.